assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -194,14 +206,37 @@ const ChecklistTab = (props) => {
     [checklist]
   );
 
+  const getOnClick = useCallback(
+    ({ elementId, pageId, elements }) => {
+      if (!elementId && !pageId && !elements) {
+        return undefined;
+      }
+      return () => {
+        if (pageId) {
+          setCurrentPage({ pageId });
+        }
+        if (Array.isArray(elements)) {
+          setSelectedElementsById({ elementIds: elements.map(({ id }) => id) });
+        } else if (elementId) {
+          setSelectedElementsById({ elementIds: [elementId] });
+        }
+      };
+    },
+    [setCurrentPage, setSelectedElementsById]
+  );
+
   const renderRow = useCallback(
-    ({ message, help, id, pageGroup }) => (
-      <Row key={`guidance-${id}`} pageGroup={pageGroup}>
+    ({ message, help, id, pageGroup, ...args }) => (
+      <Row
```

Comment:
It would be beneficial to make these focusable (with an aria label to let them know what happens if they keydown enter on it) so that keyboard users can click on them as well.